### PR TITLE
[SPARK-10371] [SQL][follow-up] fix code style

### PR DESCRIPTION
**Pull request by cloud-fan**

### CODE REVIEW

**Comment by cloud-fan** [comment]
cc @nongli @yhuai

**Comment by cloud-fan** [comment]
retest this please.

**Comment by cloud-fan** [comment]
retest this please

**Comment by nongli** [comment]
LGTM thanks!

File: sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/EquivalentExpressions.scala [hunk 1]
```diff
@@ -63,19 +61,19 @@ class EquivalentExpressions {
...
-   * Adds the expression to this datastructure recursively. Stops if a matching expression
+   * Adds the expression to this data structure recursively. Stops if a matching expression
    * is found. That is, if `expr` has already been added, its children are not added.
    * If ignoreLeaf is true, leaf nodes are ignored.
    */
   def addExprTree(root: Expression, ignoreLeaf: Boolean = true): Unit = {
     val skip = root.isInstanceOf[LeafExpression] && ignoreLeaf
-    if (!skip && root.deterministic && !addExpr(root)) {
+    if (!skip && !addExpr(root)) {
      root.children.foreach(addExprTree(_, ignoreLeaf))
```
Comment (yhuai) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/EquivalentExpressions.scala]: indent one more space? I will check this when I merge it.

**Comment by yhuai** [comment]
Thanks. Merging to master and branch 1.6.

**Pull request closed without merging**